docs: propose AEGIS import and selector

Konfekt opened the pull request:
As an aside, I agree that the idea of both factors being on the same device seems to render the whole point of two-factor moot, but that's common practice on your mobile phone, so why not offer the remaining computer users a similar convenience? Justification I heard on the mobile side being that factors needs to be interpreted more broadly / generously, say the storage location (albeit on the same device). 

Thank you very much for this extension, in any event!

Konfekt committed:
docs: add Aegis migration and paste secrets instructions

kousu commented:
Hi :) . Did you maybe squash away something in https://github.com/Konfekt/select-pass-otp ? I don't see `aesgis2pass` nor `select-secret.sh` in there.

As for rendering two-factor moot: of confidentiality, integrity and availability, I see pass-otp as mostly addressing _availability_.  Sites want me to use it because they want to enforce integrity. I want to use it because I want to have those sites available. But over plain passwords at least using it never sends the secret anywhere; I would be down if sites *only* let me log in with OTP (maybe longer 12 digit OTP).

Konfekt committed:
Remove Aegis Authenticator migration instructions

Removed instructions for migrating from Aegis Authenticator and updated the reference for the selector.

Konfekt commented:
> But over plain passwords at least using it never sends the secret anywhere

Good and essential point

> Did you maybe squash away something

Indeed, discovery of https://github.com/roddhjav/pass-import made [aegis importer](https://gist.github.com/Konfekt/12157b236ae83037de35b144488a71e3) redundant. Select-Pass now became a [pass extension](https://github.com/Konfekt/select-pass-otp);
